### ISSUE TITLE
Setup.sql file missing

### ISSUE THREAD

**Issue by daviojm** [OzWookiee/SMDR-Logger#1]
Hi there, do you still have the setup.sql file for MySql that seems to be missing from your repository on this? It's been 5 years, so I won't hold my breath. :D

**Comment by OzWookiee** [comment]
OMG I completely forgot I had this... let me quickly review

**Comment by OzWookiee** [comment]
Are you using Avaya IP Office (what this is designed for)? If so I now use a local elasticsearch/logstash/kibana to manage and visualize which is 1000% better than this very basic logging. Be happy to share my logstash config if you like, otherwise I might be able to cobble together the SQL setup file as the original files are long gone from my local PCs

**Comment by daviojm** [comment]
Yeah, I'm using Avaya IP Office. I was going to recreate the fields based on the values, but was hoping you might have the sql file still to save some time. :)

Never heard of elasticsearch/logstash. Reading up on it, it looks 1000% cooler than your basic logging. If you're ok with sharing your config, no problem. I'll read up more on logstash.

**Comment by OzWookiee** [comment]
Righto, I'll go through it when I get to work in an hour.

**Comment by OzWookiee** [comment]
OK Here we go: https://github.com/OzWookiee/avaya-smdr-elasticstack

**Comment by daviojm** [comment]
Thanks a lot. Appreciate it!

**Comment by OzWookiee** [comment]
Let me know how you get on.
